### ISSUE TITLE
4.0.5: after moving message, the focus skips over the next message 

### ISSUE THREAD

**Issue by mewalter** [opto/nostalgy-xpi#223]
Windows 10
Thunderbird 115.3.2 (32-bit)
Nostalgy 4.0.5

After moving a message to a different folder, for my installation, the focus skips over the next message and goes to the message after the next.

**Comment by opto** [comment]
Interesting. Not here: win 11, 64 bit.

anything in error console (ctrl-J)?

**Comment by mewalter** [comment]
Nothing there with ctrl-J. I did have Quick Move and tbkeys-lite installed, but removed them in case they were interfering. Only other add-on is QuickText, but after disabling it and restarting, no change. 

I could upgrade to 64 bit later in the week. No big deal with this. Just need to use up-arrow to get to what would have been the next message.

**Comment by sphh** [comment]
I experience the same behaviour as @mewalter describes.

Linux Mint 21.2 Cinnamon
Thunderbird 115.3.1 (64-bit)
Nostalgy 4.0.5
Messages sorted by date with the newest message at the bottom
Classic View (in case that makes a difference)

**Comment by opto** [comment]
yes, the function doing the original action is gone - now it is somewhere else with a name unknown in a class unknown -- one of the many searching situations TB 115 poses to addon developers

**Comment by sphh** [comment]
I do not know, if this helps, but I just noticed that clicking the `Junk` button moves the selected mail to the Junk folder and goes to the message after the next (skipping the next message).

**Comment by opto** [comment]
v4.0.9 or later (coming)
